build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
     ext.kotlin_ktx_version = '1.2.0'
     ext.wordPressUtilsVersion = '1.30.3-beta.3'
     ext.detektVersion = '1.15.0'
-    ext.gutenbergMobileVersion = 'v1.52.0'
+    ext.gutenbergMobileVersion = '3437-214e7b5314c4a30fc662493597a166aa00a957e2'
```

Comment:
> Should this revert back to pinned version before merging ?

@jhnstn ultimately it will need to be updated to be the merge commit of wordpress-mobile/gutenberg-mobile#3437 into `develop` of `gutenberg-mobile`. An example of this can be found [here](https://github.com/wordpress-mobile/WordPress-Android/pull/14528/files). 

`ext.gutenbergMobileVersion` is currently set to `<PR-number>-<commit-sha>` to allow integration/testing of the unmerged work within wordpress-mobile/gutenberg-mobile#3437. More information on this practice can be found in [WPAndroid Integration](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/WPAndroid%20Integration.md).

During the release process, `ext.gutenbergMobileVersion` is set to the appropriate tag within `gutenberg-mobile`.